specification/schemas/Report.json

```diff
@@ -1,149 +1,17 @@
-                  "items": {
-                    "type": "string",
-                    "format": "uuid",
-                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
-                    "example": "35eddf50-1d84-47a3-8479-6bfda729cd99"
-                  }
-                },
-                "organizations": {
-                  "type": "array",
-                  "description": "Array of organization ids to filter included shipments by.",
-                  "items": {
-                    "type": "string",
-                    "format": "uuid",
-                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
-                    "example": "9cdf86e8-333f-4ed9-bb31-4935c780c947"
-                  }
-                },
-                "carriers": {
-                  "type": "array",
-                  "description": "Array of carrier ids to filter included shipments by.",
-                  "items": {
-                    "type": "string",
-                    "format": "uuid",
-                    "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
-                    "example": "be7f6752-34e0-49a1-a832-bcc209450ea9"
-                  }
-                },
-                "current_status_code": {
-                  "type": "array",
-                  "description": "Comma separated string of status codes to filter by. Only shipments with the a current status that corresponds with one of the status codes will be in the report.",
-                  "items": {
-                    "type": "string",
-                    "example": "shipment-delivered"
-                  }
-                },
-                "service_code": {
-                  "type": "array",
-                  "description": "Comma separated string of service codes to filter by. Only shipments with a service that corresponds with one of the service codes will be in the report.",
-                  "items": {
-                    "type": "string",
-                    "example": "dpd-next-day"
-                  }
-                },
-                "recipient_address": {
-                  "type": "object",
-                  "required": [
-                    "country_code"
-                  ],
-                  "properties": {
-                    "country_code": {
-                      "$ref": "#/components/schemas/CountryCode"
-                    }
-                  }
-                },
-                "sender_address": {
-                  "type": "object",
-                  "required": [
-                    "country_code"
-                  ],
-                  "properties": {
-                    "country_code": {
-                      "$ref": "#/components/schemas/CountryCode"
-                    }
-                  }
-                }
-              }
-            },
             "name": {
               "type": "string",
               "example": "2020 Q1 UK shipment report"
```

Comment:
I think the count endpoint should not crash when a name is sent with the report, because it is a valid property on a `Report` resource. Let's move the `name` to the `BaseReport` file as well and keep it optional.

Reply:
I see, making the field optional will fix the issue indeed.

Reply:
@M4tini fixed!